packages/gdrcopy.spec

```diff
@@ -150,9 +150,19 @@ cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/Makefile $RPM_BUILD_ROOT%{usr
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/nv-p2p-dummy.c $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}/
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/dkms.conf $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}
 
+%if 0%{!?suse_version:1}
 # Install gdrdrv service script
 install -d $RPM_BUILD_ROOT/etc/init.d
 install -m 0755 $RPM_BUILD_DIR/%{name}-%{version}/init.d/gdrcopy $RPM_BUILD_ROOT/etc/init.d
+%else
+mkdir -p $RPM_BUILD_ROOT/etc/modprobe.d
+cat <<"EOF" > $RPM_BUILD_ROOT/etc/modprobe.d/50-gdrdrv.conf
+#options gdrdrv dbg_enabled=1
+#options info_enabled=1
+install gdrdrv PATH=$PATH:/bin:/usr/bin; /sbin/modprobe --ignore-install gdrdrv $CMDLINE_OPTS && rm -f /dev/gdrdrv && mknod -m 666 /dev/gdrdrv c $(gawk '/gdrdrv/{printf"%%s",$1}' /proc/devices) 0 && chgrp video /dev/gdrdrv
```

Comment:
Why do you do `chgrp video /dev/gdrdrv` here? Is this the most appropriate group in SLE? Is it guaranteed to always be available?

Reply:
On other platforms such as Ubuntu, `/dev/nvidia<N>` are set to `root:root` with 666 permission. We do the same with `/dev/gdrdrv`. I am not sure about on SUSE, but I suggest that you use the same permission and group as `/dev/nvidia<N>`. Note that users need both read and write permissions in order to access gdrdrv. In this case, 664 is the same as 660.

Reply:
On SUSE, `/dev/nvidia*`  is `root:video` and 660. I will change the /dev/gdrdrv permissions to 660 as well.